src/dmd/objc.d

```diff
@@ -89,11 +89,11 @@ struct ObjcSelector
     extern (C++) static ObjcSelector* create(FuncDeclaration fdecl)
     {
         OutBuffer buf;
-        size_t pcount = 0;
         TypeFunction ftype = cast(TypeFunction)fdecl.type;
         const id = fdecl.ident.toString();
+        const nparams = ftype.parameterList.length;
         // Special case: property setter
-        if (ftype.isproperty && ftype.parameterList.parameters && ftype.parameterList.parameters.dim == 1)
+        if (ftype.isproperty && nparams == 1)
```

Comment:
Strictly speaking, this is a functional change, but I think it's a fix (in case the single unexpanded param is an empty tuple).

Reply:
> in case the single unexpanded param is an empty tuple

How would that D code look like?

Reply:
```D
void foo(T...)(T params) @property { assert(params.length == 0); }
void bar() { foo(); }
```

Reply:
I see. Objective-C methods cannot be templates. So perhaps this change will not be noticeable.

Reply:
Non-templated (but probably hardly to be found in the wild):
```D
alias AliasSeq(T...) = T;
void foo(AliasSeq!() params) @property { assert(params.length == 0); }
```

Reply:
Yeah, I think we're ok.